doc/bucket-class-crd.md

```diff
@@ -72,3 +78,248 @@ status:
       lastTime: "2019-06-04T13:05:35.473Z"
       troubleshooting: "https://github.com/noobaa/noobaa-core/wiki/Bucket-class-finalizer-troubleshooting"
 ```
+
+# Example
+
+Here is an example of BucketClass CR when running different BucketClass configurations:
+
+Single tier, single backingstore, placement spread:
```

Comment:
Minor suggestion: (note the hyphenated backing-store and capitalize placement value) - same for others below

Single tier, single backing-store, placement Spread: